src/Tribe/Tickets_View.php

```diff
@@ -1144,7 +1163,11 @@ public function get_tickets_block( $post, $echo = true ) {
 				add_filter( 'tribe_tickets_order_link_template_already_rendered', '__return_true' );
 			}
 
-			return $before_content . $template->template( 'v2/tickets', [], $echo );
+			$rendered_content = $before_content;
+			$rendered_content .= $template->template( 'v2/tickets', [], $echo );
+			$rendered_content .= $template->template( 'blocks/attendees', [], $echo );
```

Comment:
Should we check if the settings for having the attendees listed is ON before concatenating?